doc/development/deprecations.rst

```diff
@@ -98,14 +98,20 @@ Pending deprecations
       qml.RX(0.2, wires=1)
       qml.expval(H)
 
-    tapes, fn = qml.transforms.hamiltonian_expand(tape)
+    tapes, fn = qml.transforms.split_qscript(tape)
 
 * ``qml.transforms.make_tape`` has been deprecated, and usage will now raise a warning.
   Instead, use ``qml.tape.make_qscript``.
 
   - Deprecated in v0.28
   - Will be removed in v0.29
 
+* ``qml.transforms.hamiltonian_expand`` has been deprecated. Users are redirected to the new
```

Comment:
Is `sum_expand` not also deprecated, or is there other uses to it?

Reply:
Good catch! It is also deprecated. Added deprecation warning 👍